quidel/delphi_quidel/pull.py

```diff
@@ -330,7 +330,7 @@ def pull_quidel_data(params):
         datetime.datetime
             the last date of the report
     """
-    cache_dir = params["cache_dir"]
+    cache_dir = params["input_cache_dir"]
```

Comment:
```suggestion
    cache_dir = params["indicator"]["input_cache_dir"]
```
?

Reply:
oh wait, i see params is passed in as `params["indicator']` elsewhere. disregard

Reply:
what do you think about renaming `params` -> `indicator_params` or something?

Reply:
That would create a lot of code diffs for what would otherwise be a drop-in change. Ideally these PRs don't change anything about the indicator functionality, and figuring out what params to pass them is the job of the Indicator Runner or some other wrapper.

Reply:
:+1: that's fine with me. for clarity I was just referring to the local scope of pull_quidel_data, not renaming params everywhere.